example/thirdparty/ollama-js/build.mill

```diff
@@ -0,0 +1,318 @@
+package build
```

Comment:
This should have a paragraph at the top of the file summarizing what this example is about, so people can read it before they dive straight into the code

Reply:
The generated `.adoc` from this should also be included in a new page `Javascript Build Examples` under the `Building Javascript with Mill` section of the docsite